src/main/kotlin/com/hibob/academy/resource/ownerResource.kt

```diff
@@ -0,0 +1,44 @@
+package com.hibob.academy.resource
+
+import com.hibob.academy.types.Pets
+import jakarta.ws.rs.*
+import jakarta.ws.rs.core.Response
+import org.springframework.stereotype.Controller
+import jakarta.ws.rs.core.MediaType
+import org.springframework.web.bind.annotation.RequestBody
+
+@Controller
+@Path("api/owners")
+@Produces(MediaType.APPLICATION_JSON)
+class OwnerResource {
+
+    @POST
+    @Path("/{Id}/type")
```

Comment:
I think you might have forgot to fix this route. When creating a pet you don't usually decide on the ID.